### ISSUE TITLE
SonarLint - Silently fails to detect issues if sonar.oe.databases is wrong

### ISSUE THREAD

**Issue by cverbiest** [Riverside-Software/sonar-openedge#654]
Sonarlint Eclipse plugin show 0 issues for source containing issues

```
Starting SonarLint for Eclipse 3.6.1008
SonarLint analysis of file /gal_smartdev/src/com/galloo/server/xgcomm/Weegticket/WeegbonhoofdDataAccess.cls...
Found 0 issue(s)
```

sonar.oe.databases value did not contain the databases needed for this class.
I could reproduce this by setting sonar.oe.databases to bad.df.
I could not find any logging showing an issue with the database connection

**Comment by movedoa** [comment]
This could be https://github.com/Riverside-Software/sonar-openedge/issues/650

**Comment by gquerret** [comment]
I confirm the behavior, which is identical in the batch scanner:
* Invalid DF files display a stack trace in the log output, but that doesn't interrupt the analysis. In SonarLint, analysis log is not displayed by default, so this error can go unnoticed
* As the DB schema is not available, the code analysis can't be completed, but once again the stack trace is not displayed. And this is reported as an analysis error, so previous lint issues are not removed, so this can be even more difficult to notice.

Additional log messages will be added.

**Comment by gquerret** [comment]
Fixed in develop
